src/data/text/item_descriptions.h

```diff
@@ -3907,3 +3907,8 @@ static const u8 sLustrousGlobeDesc[] = _(
     "A large, glowing gem\n"
     "that lets Palkia\n"
     "change form.");
+
+static const u8 sBerserkGene[] = _(
+    "Boosts Attack\n"
+    "but causes\n"
+    "confusion.");
```

Comment:
```suggestion
    "Sharply boosts\n"
    "Attack, but causes\n"
    "lasting confusion.");
```

Reply:
this is the official description in the gsc games. (see bulbapedia), If its change is still wanted I am happy with it.

Reply:
I'm pretty sure it was set like that due to character limitations. Thanks to the extra space, we can go a little bit more descriptive :)